### PR TITLE
Expose sign and verify methods

### PR DESCRIPTION
**Pull request by bkeepers**
I needed a way to easily generate/verify the webhook signature for some proper integration tests in @probot. This exposes a `sign` and `verify` method on the webhook handler that can be used to generate and verify the signatures for the given webhook secret.

Usage:

```js
const createHandler = require('github-webhook-handler')
const handler = createHandler({ path: '/webhook', secret: 'myhashsecret' })

const sig = handler.sign({foo: 'bar'})
handler.verify(sign, {foo: 'bar'})
```

### CODE REVIEW

**Review by gr2m — CHANGES_REQUESTED** [review]
needs tests for the new public APIs. Let me know if you want me to look into it :)

**Comment by bkeepers** [comment]
Updated tests to use `.sign`, and added tests for `sign`+`verify`.

File: test.js [hunk 1]
```diff
@@ -202,7 +195,7 @@ test('handler accepts a signed blob', function (t) {
     , req  = mkReq('/')
     , res  = mkRes()
 
-  req.headers['x-hub-signature'] = signBlob('bogus', json)
+  req.headers['x-hub-signature'] = h.sign(json)
```
Comment (gr2m) [inline test.js]: nice 👍

**Review by gr2m — APPROVED** [review]
:shipit:

**Pull request merged**